### PR TITLE
Add /visitors endpoint

### PR DESCRIPTION
**Pull request by thewheat**
Add support for the visitors endpoint

https://developers.intercom.io/reference#visitors

### CODE REVIEW

**Comment by Skaelv** [comment]
LGTM ! thks @thewheat

File: README.md [hunk 1]
```diff
@@ -177,6 +177,44 @@ var conversion = {
...
 
+## Visitors
+
+```node
+// Update a visitor by id
+client.visitors.update({ id: '5435345', email: 'wash@serenity.io' }, callback);
+```
+
+```node
+// Find visitor by id or user_id
```
Comment (Skaelv) [inline README.md]: `listBy` didn't really make sense here as user_id is a unique identifier.
I've updated the `find` method to handle `user_id` and `id`

**Comment by Skaelv** [comment]
@theweat small update on that one before merging
Can you give me a 👍  ?

**Comment by choran** [comment]
👍

**Pull request merged**